src/js/chapter-factory.js

```diff
@@ -1,56 +1,87 @@
-import { makeDraggable } from "./drag-drop";
+import { makeDraggable, makeDroppable } from "./drag-drop";
 
```

Comment:
Chapter factory was added and i changed the way how images are processed - now we can add class to the images when they are passed to Chapter factory as arguments. Also changed methods names and removed render method.